docs/runtime-contract.md

```diff
@@ -470,7 +470,7 @@ Seccomp provides a mechanism for further restricting the set of linux syscalls
 permitted to the processes running inside the container environment. A seccomp
 sandbox MAY be enforced by the platform operator; any such application profiles
 SHOULD be configured and applied in a consistent mechanism outside of the
-container specification. As the seccomp policy may be part of the platform
+container specification. As the seccomp policy might be part of the platform
```

Comment:
Why not 'MAY'?

Reply:
I may have read it wrong, but I didn't think the "may" here was our way of saying "you can do this if you want" but rather it was trying to explain that something this thing happens and we're not really making a statement as to whether it's ok or not.  Was it intended to introduce a constraint or was it just explaining the state of the world?

Reply:
I think we want to say that it is okay; sharing a kernel is sufficiently exciting from a security perspective that we should be clear that enabling defense mechanisms here is prudent (though not required -- something like Kata Containers or gvisor has different mechanisms for protection).

Reply:
if we want MAY in there, then I think we need to remove the "As the..." part, because then it reads kind of weird as a normative statement. See what you think of the new text